tests/unit-tests/dds/DCPS/run_test_coverage.pl

```diff
@@ -0,0 +1,79 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
```

Comment:
I'm not entirely sure what script file is for, but I don't think you needed to copy it.